stdlib/src/os/os.mojo

```diff
@@ -244,3 +244,53 @@ fn abort[
         print(message, flush=True)
 
     return abort[result]()
+
+
+# ===----------------------------------------------------------------------=== #
+# remove/unlink
+# ===----------------------------------------------------------------------=== #
+fn remove(path: String) raises:
+    """Removes the speficied file.
+
+    Args:
+      path: The path to the file.
+
+    """
+    var error = external_call["unlink", Int8](path._as_ptr())
+
+    if error != 0:
+        # TODO get error message, the following code prints it
```

Comment:
**Suggestion** Do you mind filing a GitHub issue to support capturing the error message on failure and link to it here in the code?  This will help others who are interested in improving this to be able to cross-reference things.

Reply:
I'll wait for the merge to create the issue, to be able to reference the code in the nightly branch